src/components/index.js

```diff
@@ -0,0 +1,14 @@
+export {
+  CardContent,
+  CardImage,
+  CardList,
+  CardWrapper,
+  OnlyCard,
+} from "./Card";
+
+export { FolderAdd, FolderHeader, FolderList, FolderOption } from "./Folder";
```

Comment:
P3:
특정 파일에 잇는 모든 export 를 내보내실때는 아래처럼도 가능합니다.

```suggestion
export * from "./Folder";
```